src/rpc/blockchain.cpp

```diff
@@ -1516,8 +1512,14 @@ UniValue getchaintxstats(const JSONRPCRequest& request)
         }
     }
 
+    blockcount = std::min(blockcount, pindex->nHeight - 1);
```

Comment:
@promag Thank you for you review.
I added a commit that throws a new RPC_INVALID_PARAMETER if block's height below 2.